packages/cloud-react/lib/recipes/AccountCard/index.tsx

```diff
@@ -0,0 +1,179 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import { JSX, useEffect, useState } from "react";
+
+import { Grid } from "../../base/structure/Grid";
+import { Card } from "../../base/structure/Card";
+import { GridJustify, GridSizes } from "../../base/types";
+import { PolkadotIcon } from "../../icons/PolkadotIcon";
+import { valEmpty } from "../../utils";
+
+import "@polkadot-cloud/core/css/recipes/AccountCard/index.css";
+
+interface AccountCardProps {
+  name?: string;
+  address?: string;
+  fontClass?: string;
+  ellipsis?: boolean;
+  ellipsisAmount?: number;
+  icon?: IconProps;
+  extraComponent?: ExtraComponentProps;
+}
+
+const cut = (str: string, amount = 4) => {
+  // having an amount less than 4 is a bit extreme so we default there
+  if (amount <= 4) {
+    return str.slice(0, 4) + "..." + str.slice(-4);
+  }
+  // if the amount requested is in a "logical" amount - meaning that it can display the address
+  // without repeating the same information twice - then go for it;
+  if (amount <= str.length / 2 - 3) {
+    return str.slice(0, amount) + "..." + str.slice(-amount);
+  }
+  // else, the user has been mistaskenly extreme, so just show the maximum possible amount
+  return (
+    str.slice(0, str.length / 2 - 3) + "..." + str.slice(-(str.length / 2 - 3))
+  );
+};
+
+export interface IconProps extends CompParams {
+  canCopy?: boolean;
+  position?: "left" | "right";
+}
+
+export interface ExtraComponentProps extends CompParams {
+  component?: JSX.Element;
+  position?: "left" | "right";
+}
+
+export interface CompParams {
+  size?: GridSizes;
+  justify?: GridJustify;
+}
+
+export const AccountCard = ({
+  address,
+  fontClass,
+  ellipsis = false,
+  ellipsisAmount = 7,
+  icon,
+  extraComponent,
+}: AccountCardProps) => {
+  const fontClasses: string[] = [
+    "font-classes",
+    valEmpty(fontClass, fontClass),
+    valEmpty(ellipsis, "ellipsis"),
+    "main-text",
```

Comment:
I'm not sure exactly why, but `valEmpty` is resulting in spaces here:
<img width="332" alt="Screenshot 2023-08-25 at 21 57 14" src="https://github.com/paritytech/polkadot-cloud/assets/13929023/e5659d0e-7071-4792-8335-62c2a1a92b2c">

Reply:
Fixed. Thank you